# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/HTMLMediaElement.h
# Commit: a8a44fb117cb
# Full Hash: a8a44fb117cb6baef601c7ba96b73ac9066d3272
# Author: alwu <alwu@mozilla.com>
# Date: 2020-07-24 09:32:06
# Regressor Bug: 1653390
# File Overlap Count: 1
# Description:
#   Bug 1653390 - part5 : remove 'StopListeningMediaControlKeyIfNeeded()'. r=bryce
#   
#   First, rename `Stop()` to `StopIfNeeded()`.
#   
#   Second, we have already have a check [1] in `StopIfNeeded()` to ensure that that function is only workable when the listener starts, so we don't need `StopListeningMediaControlKeyIfNeeded()` and can call `StopIfNeeded()` directly.
# ==============================================================================

diff -r 3bb773642909 -r a8a44fb117cb dom/html/HTMLMediaElement.h
--- a/dom/html/HTMLMediaElement.h	Thu Jul 23 21:33:23 2020 +0000
+++ b/dom/html/HTMLMediaElement.h	Thu Jul 23 21:33:31 2020 +0000
@@ -1930,7 +1930,6 @@
   // We use MediaControlKeyListener to listen media control key, by which we
   // would play or pause media element.
   void StartListeningMediaControlKeyIfNeeded();
-  void StopListeningMediaControlKeyIfNeeded();
   RefPtr<MediaControlKeyListener> mMediaControlKeyListener;
 
   // Return true if the media element is being used in picture in picture mode.
